src/main/java/jude/Main.java

```diff
@@ -0,0 +1,48 @@
+package jude;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+//@@author cheeheng-reused
+//Reused from https://se-education.org/guides/tutorials/javaFxPart4.html with minor
+//modifications.
+
+/**
+ * A GUI for Jude the chatbot using FXML.
+ */
+public class Main extends Application {
+    private final Jude jude = new Jude("data/tasks.txt");
```

Comment:
For easier modification, perhaps you could consider refactoring 'data/tasks.txt' into a constant TASKS_LIST or something outside the code just for easier readability/editing in the future!